doc/modules/preprocessing.rst

```diff
@@ -240,10 +240,63 @@ Centering kernel matrices
 -------------------------
 
 If you have a kernel matrix of a kernel :math:`K` that computes a dot product
-in a feature space defined by function :math:`\phi`,
-a :class:`KernelCenterer` can transform the kernel matrix
-so that it contains inner products in the feature space
-defined by :math:`\phi` followed by removal of the mean in that space.
+in a feature space defined by a function :math:`\phi(\cdot)`, a
```

Comment:
```suggestion
in a feature space (possibly implicitly) defined by a function :math:`\phi(\cdot)`, a
```

Reply:
this suggestion would imply a reformatting of the paragraph to fit in 80 columns. Maybe it's not worth it. Feel free to resolve without fixing ;)